src/platform/test.rs

```diff
@@ -412,6 +414,40 @@ fn receiver_set() {
 }
 
 #[test]
+#[cfg(not(any(feature = "force-inprocess", target_os = "windows", target_os = "android")))]
+fn receiver_set_eintr() {
+    let (server, name) = OsIpcOneShotServer::new().unwrap();
+    let (tx1, rx1) = platform::channel().unwrap();
+    let mut rx_set = OsIpcReceiverSet::new().unwrap();
+    let rx_id = rx_set.add(rx1).unwrap();
+    let child_pid = unsafe {
+        fork(|| {
+            // Let the parent know we're ready
+            let tx0 = OsIpcSender::connect(name).unwrap();
+            tx0.send(b"\x00\x00\x00\x00", vec![], vec![]).unwrap();
+            // Send the result of the select back to the parent
+            let result = rx_set.select().unwrap();
+            let mut result_iter = result.into_iter();
+            let (id, data, _, _) = result_iter.next().unwrap().unwrap();
+            assert_eq!(rx_id, id);
+            assert_eq!(vec![0, 0, 0, 0], data);
```

Comment:
`assert_eq!(b"\x00\x00\x00\x00".as_ref(), &*data)` I like this best. The update uses this and uses more descriptive strings.